tests/test_utils.py

```diff
@@ -364,27 +364,27 @@ def test_augment_diagonal_undirected(self):
         expected[0, 0] = 2.0 / 4
         expected[1, 1] = 3.0 / 4
         expected[2, 2] = 3.0 / 4
-        expected[3, 3] = 2.0 / 4
+        expected[3, 3] = 3.0 / 4
```

Comment:
Please do not change these tests. These exist so that if they fail, then you broke the code somehow.

Reply:
will fix this soon.

Reply:
I got the reason why I can't fix these out. Itried to fix this by comit test_utils.py from a clean one cloned from master graspy. I found now the one in master graspy is just this one. I don't know why it mark here as changed.
See:
https://github.com/neurodata/graspy/blob/master/tests/test_utils.py
```

Reply:
you need to remove this commit from this PR

Reply:
I keep have the error trying to use 'revert' command to reverse this commit.
The error information :

Reply:
fatal error : revert is not possible because you have unmerged files

Reply:
I really have no idea on how to fix this, I have tried some advice from stackoverflow but I don't think they worked.

Reply:
For now I can promise you these changes are harmless because you can check the link:
https://github.com/neurodata/graspy/blob/master/tests/test_utils.py
The file in the current version looks exactly like this.
I do feel sorry for putting this issue in such a dilemma because of my incorrect act of deleting and re-adding files two month ago. Currently I just jave two ideas on how to deal with it :One is merging it as it is ,the other is close this pr and start a new clean on directly from my fork.

Reply:
If you have any better solution I would really really appreciate it.